k8s/prometheus-operator/README.md

```diff
@@ -129,6 +136,28 @@ Configure the container image:
 export IMAGE_OPERATOR="marketplace.gcr.io/google/prometheus-operator"
 ```
 
+#### If using GKE on-prem
+Test Google Container Registry (GCR) accessibility,
+```
+kubectl -n "${NAMESPACE}" run test-image --image="marketplace.gcr.io/google/ubuntu:latest" -- sleep 60000 1>/dev/null && kubectl -n "${NAMESPACE}" get pod test-image -o=go-template='{{$output := "Failed to pull image from GCR"}}{{range .status.containerStatuses}}{{if eq .ready true}}{{$output = (print "Successfully pulled image from GCR. ")}}{{end}}{{end}}{{printf $output}}{{"\n"}}' && kubectl -n "${NAMESPACE}" delete pod test-image 1>/dev/null
+```
+If you do not see commmand outputs the message
+> Successfully pulled image from GCR.
+
+
+, create a gcloud service account and download the json key file. 
```

Comment:
I am not super clear on the "create a service account here"

For GKE on-prem, we already request customers to create service accounts, and those service account was responsible for pulling Anthos images as well. So we may want to double check with the existing documents (https://cloud.google.com/anthos/gke/docs/on-prem) to have a uniform story in this user guide as well.

Reply:
Now the doc is under the assumption that users have gone through the onprem doc above (and that doc is included in README)